modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaRow.es.js

```diff
@@ -586,14 +386,20 @@ class CriteriaRow extends Component {
 					connectDragPreview(
 						<div className={classes}>
 							{editing ? (
-								this._renderEditContainer({
-									error,
-									propertyLabel,
-									renderEmptyValuesErrors,
-									selectedOperator,
-									selectedProperty,
-									value,
-								})
+								<CriteriaRowEditable
+									connectDragSource={connectDragSource}
```

Comment:
You don't need to pass the connectDragSource prop to the component. It will have access to it because of the DndProvider.